macros/CSS_Ref.ejs

```diff
@@ -13,12 +13,12 @@
        defaults to ['standard'])
 */
 
-var propertiesUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/properties.json";
-var atRulesUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/at-rules.json";
-var selectorsUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/selectors.json";
-var typesUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/types.json";
-var syntaxesUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/syntaxes.json";
-var unitsUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/units.json";
+var propertiesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/properties.json";
+var atRulesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/at-rules.json";
+var selectorsUrl = "https://raw.githubusercontent.com/mdn/data/master/css/selectors.json";
+var typesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/types.json";
+var syntaxesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/syntaxes.json";
+var unitsUrl = "https://raw.githubusercontent.com/mdn/data/master/css/units.json";
```

Comment:
I've confirmed that the removed links are back.  There are now two added links in dev:

* [&lt;an-plus-b&gt;](https://developer.mozilla.org/en-US/docs/Web/CSS/an-plus-b)
* [text-justify](http://developer.mozilla.org/en-US/docs/Web/CSS/text-justify)